easybuild/framework/easyblock.py

```diff
@@ -1800,6 +1803,14 @@ def prepare_step_spec(initial):
                 lambda x: x.test_cases_step(),
             ], False))
 
+        ## CHANGE TRUE TO build_option, and 
+        ## ADD build-pkg to all the configuration dicts
+        ## # if build_option('build-pkg'):
```

Comment:
see #1185 for inspiration on how to add a new build option (look for `update-modules-tool-cache` and `update_modules_tool_cache`)